reactClient/src/Components/login/LoginModal.js

```diff
@@ -1,23 +1,45 @@
 import React from "react";
 import 'bootstrap/dist/js/bootstrap.bundle.min.js'
 import 'bootstrap/dist/css/bootstrap.min.css'
+import axios from "axios";
```

Comment:
make sure you import from the instance in `utils/axios` so the base url always gets set
```suggestion
import axios from '../../utils/axiosInstance';
```